prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -382,7 +382,10 @@ fi
 
 # ---------------------------------------------------------------------------------------------------------------------
 # Calibration workflows
-workflow_has_parameter CALIB && { source $MYDIR/calib-workflow.sh; [[ $? != 0 ]] && exit 1; }
```

Comment:
In bash you can just do
```
workflow_has_parameter CALIB && { source ${CALIB_WF:-$MYDIR/calib-workflow.sh}; [[ $? != 0 ]] && exit 1; }
```
to provide a default parameter for an empty variable.